docs/guides/sqd-intro.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Sample-based quantum diagonalization
+description: Overview of the Sample-based quantum diagonalization (SQD) workflow
+---
+
+# Sample-based quantum diagonalization (SQD)
+
+Sample-based quantum diagonalization (SQD) is a technique for finding eigenvalues and eigenvectors of quantum operators, such as a the Hamiltonian of a quantum system, using quantum and distributed classical computing together.
+
+Classical computing is used to process samples obtained from a quantum processor, and to project and diagonalize a target Hamiltonian in a subspace spanned by them.This allows SQD to be robust to samples corrupted by quantum noise and deal with large Hamiltonians, such as chemical systems with millions of interacting terms, beyond the reach of exact diagonalization methods.
+
+The SQD tool can target Hamiltonians expressed as linear combinations of Pauli operators, or second-quantized fermionic operators. The input samples are obtained by quantum circuits defined by the user, which are believed to be good representations of eigenstates (e.g. the ground state) of a target operator. The convergence rate of SQD as a function of the number of samples improves with the sparseness of the target eigenstate.
```

Comment:
we avoid `e.g`, `i.e.`, and so forth - can we replace with "such as" or "for example" here?

Reply:
Fixed now!